package/crds/project.launchdarkly.com_environments.yaml

```
@@ -173,6 +173,80 @@ spec:
                       (String) The LaunchDarkly project key. A change in this field will force the destruction of the existing resource and the creation of a new one.
                       The LaunchDarkly project key. A change in this field will force the destruction of the existing resource and the creation of a new one.
                     type: string
+                  projectKeyRef:
+                    description: Reference to a Project in project to populate projectKey.
```

idk if this is non-generated code but should this say 
```suggestion
                    description: Reference to a project in environment to populate projectKey.
```
??

---

This is generated so I don't think we can just change it like this. Pretty strange though.